### ISSUE TITLE
Add in Caliber phenotypes identified during validation work

### ISSUE THREAD

**Issue by Hannah-Davies** [SwanseaUniversityMedical/concept-library#703]
add in the additional pheontypes identified during validation work

Use new template to upload

see email / word doc
[Re Guidance needed for validation information of phenotypes.docx](https://github.com/SwanseaUniversityMedical/concept-library/files/8715319/Re.Guidance.needed.for.validation.information.of.phenotypes.docx)

**Comment by Hannah-Davies** [comment]
@shahzadmumtaz22 is there any update on this please?

**Comment by shahzadmumtaz22** [comment]
@Hannah-Davies No progress yet. Right now I am working on some other projects and have a plan to pick this up by the end of June. I hope that should be okay.

**Comment by shahzadmumtaz22** [comment]
Hi Hannah and @ieuans, 

By the end of February, we discussed an issue of rending images when uploading phenotypes through YAML format. Ieuan pointed out that he will have to upload images to a static folder and then will send me the link to use within YAML files so that when the phenotypes are uploaded the images are also rendered. 

I emailed Ieuan all the images but have not heard back though I was also busy the whole month of March and therefore did not chase him up. I have just sent a reminder message to Ieuan for uploading images to a static folder so I can upload phenotypes with images in place.
